python/kafka_stream.py

```diff
@@ -0,0 +1,81 @@
+import mgp
+import json
+from kafka.producer import KafkaProducer
+from typing import Dict
+
+
+producers_by_name: Dict[str, KafkaProducer] = {}
+topics_by_name: Dict[str, str] = {}
+
+
+@mgp.read_proc
+def create_push_stream(
+    context: mgp.ProcCtx,
+    stream_name: str,
+    topic: str,
+    config: mgp.Map,
+) -> mgp.Record(created=bool):
+
+    if not isinstance(stream_name, str):
+        raise TypeError("Invalid type on first argument!")
```

Comment:
```suggestion
        raise TypeError("Invalid type of first argument!")
```

Reply:
Will do these changes. As for Kafka, I'm not really sure how to test it since Kafka needs to have a broker in order to create a KafkaProducer :/ It requires some external system functionality, and we agreed to have it addressed with @jmatak  and @gitbuda . What do you think guys?